addons/website_mass_mailing/static/src/js/website_mass_mailing.js

```diff
@@ -35,7 +35,6 @@ publicWidget.registry.subscribe = publicWidget.Widget.extend({
             self.$('input.js_subscribe_email')
                 .val(data.email || "")
                 .prop('disabled', isSubscriber);
-            self.$target.removeClass('d-none');
```

Comment:
Mmh, indeed never questioned the fact it was hidden for no apparent reason. Should it be disabled during loading though though? Same for the button? Or is it already the case?

Reply:
Do you mean that is should be disabled by default until the RPC comes back? That's not currently the case.

Should I disable it on start and re-enable it after the RPC comes back if needed?

Reply:
> That's not currently the case.
Well, it was hidden I think? So more than the case ^^

Ideally, it should not be disabled on start, it's way to late. It should be saved disabled or with the "o_wait_lazy_js" if clicking on it while everything is not setup correctly yet could induce a traceback.

If clicking on it has no breaking effect, it is fine showing it while it does nothing and your PR is good, just a question to be sure :)

Reply:
Yeah nothing should break if you click on the button before the widget is loaded (maybe it would scroll to the top of the page because the buttons are links to '#'). I could save the buttons in a disabled state as well if you think it's useful.

Reply:
I left this die for quite a while now... It cannot be merged in 13.0 anyway, I am ok with just the d-none stuff but current snippet in 13.0 should still work. So keep the d-none class removal and add a comment :+1: